app/src/models/services/Student/Student.js

```diff
@@ -119,9 +115,15 @@ class Student {
       const student = await StudentStorage.findOneByNameAndEmail(clientInfo);
 
       if (student) {
-        return { success: true, id: student.id };
+        const id = { id: student.id };
+
+        return Student.makeResponseMsg(
+          200,
+          '해당하는 아이디를 찾았습니다.',
+          id
```

Comment:
```suggestion
        return Student.makeResponseMsg(
          200,
          '해당하는 아이디를 찾았습니다.',
          student
```
Storage의 함수 자체가 118번 라인의 객체 자체를 반환하기때문에 따로 선언해줄 필요는 없을거 같아요.